### PR TITLE
refactor the examples

### PR DESCRIPTION
**Pull request by wopeizl**
1. move the original examples into fit_a_line sub folder
2. add the ctr exampel
test=develop

### CODE REVIEW

File: example/ctr/script/k8s_tools.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (typhoonzero) [inline example/ctr/script/k8s_tools.py]: If changes to this file and `paddle_k8s` can be compatible with all jobs, can just change these files in the original place, try not to keep multiple copies of this file

File: example/ctr/README [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+CTR分布式训练
```
Comment (typhoonzero) [inline example/ctr/README:2]: better to write this using "markdown" format, and write the steps for use

File: example/ctr/script/k8s_tools.py (hunk 1 above)
Comment (wopeizl) [reply example/ctr/script/k8s_tools.py]: ok

File: example/ctr/README (hunk 2 above)
Comment (wopeizl) [reply example/ctr/README:2]: ok

File: docker/paddle_k8s [hunk 3]
```diff
@@ -62,27 +62,33 @@ check_trainer_ret() {
...
 
   if [ "${PADDLE_TRAINING_ROLE}" == "TRAINER" ] || [ "${PADDLE_TRAINING_ROLE}" == "WORKER" ]; then
-    stdbuf -oL python /root/k8s_tools.py wait_pods_running ${trainer_label} ${PADDLE_TRAINERS}
+    stdbuf -oL python /root/k8s_tools.py wait_pods_running ${trainer_label} ${PADDLE_TRAINERS_NUM}
   fi
 
-  export PADDLE_PSERVER_IPS=$(python /root/k8s_tools.py fetch_ips ${pserver_label} ${PADDLE_INIT_PORT})
-  export PADDLE_TRAINER_IPS=$(python /root/k8s_tools.py fetch_ips ${trainer_label})
+  export PADDLE_PSERVERS=$(python /root/k8s_tools.py fetch_endpoints ${pserver_label} ${PADDLE_PORT})
+  # export PADDLE_TRAINER_IPS=$(python /root/k8s_tools.py fetch_ips ${trainer_label})
```
Comment (typhoonzero) [inline docker/paddle_k8s]: why comment out this line?

File: docker/paddle_k8s [hunk 4]
```diff
@@ -62,27 +62,33 @@ check_trainer_ret() {
...
 
   if [ "${PADDLE_TRAINING_ROLE}" == "TRAINER" ] || [ "${PADDLE_TRAINING_ROLE}" == "WORKER" ]; then
     check_failed_cnt 1
     task_index=$(python /root/k8s_tools.py fetch_id ${trainer_label})
+    # download training materials for trainer task_index=0-9
```
Comment (typhoonzero) [inline docker/paddle_k8s]: download procedure can be put into train.py

File: example/ctr/ps-train/trainer.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    metadata:
+      labels: {paddle-job: fluid-ctr}
+    spec:
+      restartPolicy: "OnFailure"
+      containers:
+      - command: [paddle_k8s, start_fluid]
+        env:
+        - {name: GLOG_v, value: '0'}
+        - {name: GLOG_logtostderr, value: '1'}
+        #- {name: GLOG_vmodule, value: 'executor=4,operator=4'}
```
Comment (typhoonzero) [inline example/ctr/ps-train/trainer.yaml]: remove commented out line or add description

File: docker/paddle_k8s (hunk 3 above)
Comment (wopeizl) [reply docker/paddle_k8s]: restore

File: docker/paddle_k8s (hunk 4 above)
Comment (wopeizl) [reply docker/paddle_k8s]: done

File: example/ctr/ps-train/trainer.yaml (hunk 5 above)
Comment (wopeizl) [reply example/ctr/ps-train/trainer.yaml]: ok

**Review by typhoonzero — APPROVED** [review]

**Comment by typhoonzero** [comment]
Force merged, I'll fix the build issue on develop branch later.

**Pull request merged**